changelog.yaml

```diff
@@ -18,7 +18,45 @@
 # Do not change the order ("NEXT" needs to be on top).
 - version: "NEXT"
   date: TBD
-  
+  changes:
+
+  - title: Introduce feature flags
+    type: internal
+    impact: patch
+    description: >-

```

Comment:
We need to change all `>-` to `|-`. Don't touch the existing ones to avoid conflicts.

Reply:
Included in 1ba45c9.